cmd/frontend/frontend.go

```diff
@@ -9,6 +9,7 @@ import (
 )
 
 const FrontendDir string = "./frontend"
+const AwsRegion = "us-west-2"
```

Comment:
are `consts` from other `.go` files importable in a given `.go` file just like javascript exported constants?

Reply:
the way i've seen it, u just import the file

Reply:
yea i can just do that --  just have to make a constants.go file.

Reply:
ok

Reply:
also running into problems here. hmm there seems to be problems when trying to use a function/constant across multiple folders. I'll look into this further and circle back to it on Sunday.

Reply:
Sounds good. lmk if you are blocked! @codingwithsurya

Reply:
workss now!